integration/nemo/examples/supervised_fine_tuning/README.md

```diff
@@ -0,0 +1,190 @@
+## Supervised Fine-tuning (SFT) with NeMo
+
+In this example, we utilize NeMo's [supervised fine-tuning](https://github.com/NVIDIA/NeMo-Megatron-Launcher#515-instruction-following-via-supervised-finetuning--sft-)
+feature to showcase how to fine-tune the whole model on supervised data for learning how to follow user specified instructions. 
+
+Due to the large model size of the LLM, we use NVFlare's streaming feature to transfer the model in chunks.
+
+## Dependencies
+This example running a 1.3B GPT model requires considerable computational resources. For training 1.3B model, SFT needs ~30GB GPU memory using fp16 precision. Hence, to run three clients in parallel, we can compute the resource needs accordingly.
+The example for 3-client 1.3B GPT model was performed on two A100 GPUs with 80 GB memory each.
+
+We assume you followed the instructions [here](../../README.md#requirements) 
+to install the NeMo, NVFlare, and the NeMo-NVFlare package. 
+
+The example was tested using the [NeMo Docker container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo), 
+available with `docker pull nvcr.io/nvidia/nemo:23.02`.
+
+For downloading the pre-trained model, we use [git lfs](https://git-lfs.com).
+
+## Download the pre-trained LLM
+In this example, we use [Megatron-GPT 1.3B](https://huggingface.co/nvidia/nemo-megatron-gpt-1.3B) and [Megatron-GPT 5B](https://huggingface.co/nvidia/nemo-megatron-gpt-5B), transformer-based language models based on the GPT architecture.
+We download the model from HuggingFace using git lfs
+```
+mkdir Models
```

Comment:
This will cause a `FileNotFoundError: Can't find /workspace/Models/nemo-megatron-gpt-1.3B/nemo_gpt1.3B_fp16.nemo` later. Could download to /workspace/Models directly.

Reply:
added the info